subscriptions/tests.py

```diff
@@ -1562,6 +1563,284 @@ def test_retry_send_message_task_to_mother_text(self):
+                "previous": None,
+                "results": [{"address": "+2345059992222"}]
+            },
+            status=200, content_type='application/json',
+            match_querystring=True
+        )
+
+        # Create message sender call
+        responses.add(
+            responses.POST,
+            "http://seed-message-sender/api/v1/outbound/",
+            json={
+                "url": "http://seed-message-sender/api/v1/outbound/c7f3c839-2bf5-42d1-86b9-ccb886645fb4/",  # noqa
+                "id": "c7f3c839-2bf5-42d1-86b9-ccb886645fb4",
+                "version": 1,
+                "to_addr": "+2345059992222",
+                "to_identity": "8646b7bc-b511-4965-a90b-e1145e398703",
+                "vumi_message_id": None,
+                "content": None,
+                "delivered": False,
+                "attempts": 0,
+                "metadata": {
+                    'voice_speech_url': 'fakefilename.mp3'
+                },
+                "created_at": "2016-03-24T13:43:43.614952Z",
+                "updated_at": "2016-03-24T13:43:43.614921Z"
+            },
+            status=200, content_type='application/json'
+        )
+
+        # Create metrics call - deactivate TestSession for this
+        self.session = None
+        responses.add(
+            responses.POST,
+            "http://metrics-url/metrics/",
+            json={"foo": "bar"},
+            status=200, content_type='application/json'
+        )
+
+        responses.add(
+            responses.POST,
+            "http://metrics-url/metrics/",
+            json={"foo": "bar"},
+            status=200, content_type='application/json'
+        )
+
+        # make binarycontent
+        binarycontent_data1 = {
+            "content": "fakefilename1.mp3",
+        }
+        binarycontent1 = BinaryContent.objects.create(**binarycontent_data1)
+        binarycontent_data2 = {
+            "content": "fakefilename2.mp3",
+        }
+        binarycontent2 = BinaryContent.objects.create(**binarycontent_data2)
+        binarycontent_data3 = {
+            "content": "fakefilename3.mp3",
+        }
+        binarycontent3 = BinaryContent.objects.create(**binarycontent_data3)
+
+        # make messages
+        message_data1 = {
+            "messageset": existing.messageset,
+            "sequence_number": 1,
+            "lang": "eng_ZA",
+            "binary_content": binarycontent1,
+        }
+        Message.objects.create(**message_data1)
+        message_data2 = {
+            "messageset": existing.messageset,
+            "sequence_number": 2,
+            "lang": "eng_ZA",
+            "binary_content": binarycontent2,
+        }
+        Message.objects.create(**message_data2)
+        message_data3 = {
+            "messageset": existing.messageset,
+            "sequence_number": 3,
+            "lang": "eng_ZA",
+            "binary_content": binarycontent3,
+        }
+        Message.objects.create(**message_data3)
```

Comment:
Would it be worth popping some of this code shared across tests inside a `setUp` function or into utility functions to keep it DRY?